db/schema.rb

```diff
@@ -115,6 +126,7 @@
     t.string "distributor"
     t.string "documentation_url"
     t.string "support_url"
+    t.string "service_offering_icon_ref"
```

Comment:
This shouldn't be here, just remove this line. It is leftover from when we changed to hosting our own icons.